media-libs/libsmf/libsmf-1.3.recipe

```diff
@@ -0,0 +1,83 @@
+SUMMARY="A C library for handling SMF files"
+DESCRIPTION="LibSMF is a BSD-licensed C library for handling \
+SMF (*.mid) files. It transparently handles conversions between time \
+and pulses, tempo map handling etc. The only dependencies are C \
+compiler and glib. Full API documentation and examples are included"
+HOMEPAGE="https://sourceforge.net/projects/libsmf/"
+COPYRIGHT="2007-2008 Edward Tomasz Napieraa \
+	2013 John Stumpo"
+LICENSE="BSD (2-clause)"
+REVISION="1"
+SOURCE_URI="https://downloads.sourceforge.net/libsmf/libsmf-$portVersion.tar.gz"
+CHECKSUM_SHA256="d3549f15de94ac8905ad365639ac6a2689cb1b51fdfa02d77fa6640001b18099"
+
+ARCHITECTURES="x86_gcc2 x86 x86_64"
+SECONDARY_ARCHITECTURES="x86_gcc2 x86"
+
+PROVIDES="
+	libsmf$secondaryArchSuffix = $portVersion
+	lib:libsmf$secondaryArchSuffix = 0.0.0 compat >= 0
+	cmd:smfsh$secondaryArchSuffix = $portVersion
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libiconv$secondaryArchSuffix
+	lib:libintl$secondaryArchSuffix
+	lib:libglib_2.0$secondaryArchSuffix
+	"
+
+PROVIDES_devel="
+	libsmf${secondaryArchSuffix}_devel = $portVersion
+	devel:libsmf$secondaryArchSuffix = 0.0.0 compat >= 0
+	"
+REQUIRES_devel="
+	libsmf$secondaryArchSuffix == $portVersion base
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	devel:libglib_2.0$secondaryArchSuffix
+	"
+BUILD_PREREQUIRES="
+	cmd:aclocal
+	cmd:autoconf
+	cmd:automake
+	cmd:awk
+	cmd:libtoolize$secondaryArchSuffix
+	cmd:make
+	cmd:gcc$secondaryArchSuffix
+	cmd:pkg_config$secondaryArchSuffix
+	"
+
+BUILD()
+{
+	libtoolize --force --copy --install
+	aclocal
+	autoheader
+	automake --add-missing
+	autoconf
```

Comment:
I've checked on all possible architectures that we don't need the call to autoconf (and autoheader).
So if you could remove the line with the call to autoconf it would be great.
(Please keep, however, the cmd:autoconf in BUILD_PREREQUIRES).
Thanks!

BTW, I maybe it would be woth to replace
```
	runConfigure ./configure \
		CPPFLAGS="-D_BSD_SOURCE"
```
by
```
	runConfigure ./configure CPPFLAGS="-D_BSD_SOURCE"
```
What do you think?
It is unlikely that we will ever need to add more parameters, so keeping the runConfigure in a single line might be a good idea.
Well, since this is just a style question, I leave you the choice ;-)